src/screens/Home.js

```diff
@@ -1,31 +1,52 @@
 import { render } from '../shared/dom.js';
 import ApiProvider from '../providers/ApiProvider.js';
-import { MovieCarousel } from '../components';
-import ConsoleLogger from '../utils/ConsoleLogger.js';
+import { Button, MovieCarousel } from '../components';
+import './Home.scss';
+import { redirect } from '../index.js';
 
 const template = async (movies) => `
-    <div>
-        <div id="home-screen">This is home screen</div>
+    <div id="homeScreen" class="homeScreen">
         <template id="home-movie-tiles"></template>
-        ${await MovieCarousel({ movies: movies[0], id: 'Carousel0' })}
-        ${await MovieCarousel({ movies: movies[1], id: 'Carousel1' })}
-        ${await MovieCarousel({ movies: movies[2], id: 'Carousel2' })}
-        ${await MovieCarousel({ movies: movies[3], id: 'Carousel3' })}
+        ${Button({
+            onClick: () => redirect('/search'),
+            id: 'topMovies',
+            text: 'Top 250 movies'
+        })}
+        ${await MovieCarousel({
+            movies: movies[0],
+            id: 'topMovies'
+        })}
+        ${Button({
```

Comment:
You are trusting the API More than you should,
If like movie.results wouldnt have any item,
Or api would return an error or something, we shouldn't render nor the button, nor the carousel

And I think that we can include this button into carousel component, instead of usung two separate components here, carousel always have button above on the designs so yeah, do that 8)

Reply:
I think it's done